mriqc/workflows/diffusion/base.py

```diff
@@ -416,6 +421,102 @@ def dmri_bmsk_workflow(name="dmri_brainmask", omp_nthreads=None):
     return workflow
 
 
+def init_dmriref_wf(
+    in_file=None,
+    multiecho=False,
+    name="init_dmriref_wf",
+):
+    """
+    Build a workflow that generates reference images for a dMRI series.
+
+    The raw reference image is the target of :abbr:`HMC (head motion correction)`, and a
+    contrast-enhanced reference is the subject of distortion correction, as well as
+    boundary-based registration to T1w and template spaces.
+
+    This workflow assumes only one dMRI file has been passed.
+
+    Workflow Graph
+        .. workflow::
+            :graph2use: orig
+            :simple_form: yes
+
+            from mriqc.workflows.diffusion.base import init_dmriref_wf
+            wf = init_dmriref_wf()
+
+    Parameters
+    ----------
+    in_file : :obj:`str`
+        dMRI series NIfTI file
+    multiecho : :obj:`bool`
+        If multiecho data was supplied, data from the first echo will be selected
```

Comment:
Multiecho is only used for workflow reporting. I don't think MRIQC uses that.

```suggestion
    name="init_dmriref_wf",
):
    """
    Build a workflow that generates reference images for a dMRI series.

    The raw reference image is the target of :abbr:`HMC (head motion correction)`, and a
    contrast-enhanced reference is the subject of distortion correction, as well as
    boundary-based registration to T1w and template spaces.

    This workflow assumes only one dMRI file has been passed.

    Workflow Graph
        .. workflow::
            :graph2use: orig
            :simple_form: yes

            from mriqc.workflows.diffusion.base import init_dmriref_wf
            wf = init_dmriref_wf()

    Parameters
    ----------
    in_file : :obj:`str`
        dMRI series NIfTI file
```